pygmt/io.py

```diff
@@ -0,0 +1,36 @@
+"""
+PyGMT input/output (I/O) utilities.
+"""
+import xarray as xr
+
+
+def process_output_grid(grid_name, tmpfile_name):
+    """
+    Processes the output from the GMT API to return an xarray.DataArray if
+    ``grid_name`` matches ``tmpfile_name`` and return None if it does not.
+
+    Parameters
+    ----------
+    grid_name : str
+        The name of the output netCDF file with extension .nc to store the grid
+        in.
+    tmpfile_name : str
+        The name attribute from a GMTTempFile instance.
+
+    Returns
+    -------
+    ret: xarray.DataArray or None
+        Return type depends on whether the ``outgrid`` parameter is set:
+
+        - :class:`xarray.DataArray` if ``outgrid`` is not set
+        - None if ``outgrid`` is set (grid output will be stored in file set by
+          ``outgrid``)
```

Comment:
Suggest changing this to match https://github.com/pydata/xarray/blob/a21dead8621480b608827355b50e85af6f88f185/xarray/backends/api.py#L259-L264

```suggestion
    datarray : xarray.DataArray
        The newly created DataArray.

    See Also
    --------
    xarray.open_dataarray
```